lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/JavaBridge.java

```diff
@@ -25,6 +25,7 @@
     try {
       System.loadLibrary(CPP_BRIDGE);
     } catch (UnsatisfiedLinkError e) {
+      // TODO[JIRA REEF-383] Document/Refactor JavaBridge
```

Comment:
This fixes the last violation of EmptyBlock check which I've cleaned up in REEF-360 (I've just used the new TODO format to get rid of it). I can remove new TODO format but I'd rather fix this violation with this checkin